src/components/user/Team.jsx

```diff
@@ -1,46 +1,144 @@
+import Button from "../Button";
 import Input from "../Input";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import toast from "react-hot-toast";
+import Loading from "../Loading";
+const Team = ({ user, setUser }) => {
+  const [team, setTeam] = useState(null);
+  const [teamID, setTeamID] = useState({});
+  const [edit, setEdit] = useState(false);
+  const handleLeave = () => {
+    axios.delete("/api/members").then(() => {
+      toast("✅ Successfully leave team!");
+      setTeam(null);
+      setUser({ ...user, team: null });
+      setTeamID({});
+    });
+  };
+  const handleJoin = () => {
+    axios
+      .put("/api/members", { team: teamID.team })
+      .then(() => {
+        toast("✅ Successfully join the team!");
+        setUser({ ...user, team: teamID.team });
+      })
+      .catch(({ response }) => {
```

Comment:
```suggestion
      .catch(({ response }) => {
```

if u do response: data I think that that can save u having to write response each time